### ISSUE TITLE
SD Card issue

### ISSUE THREAD

**Issue by shouyou10** [Secure-File-Manager/Secure-File-Manager#16]
Despite the update, in my phone the problem still remains. Can you help me in this regard?
![Imagepipe_0](https://user-images.githubusercontent.com/76897897/110048031-e817f900-7d46-11eb-8f14-87bb6816b5db.jpg)

**Comment by Alfavio** [comment]
Hi @shouyou10. Thank you for the bug report! Did you selected the root of the SD card?

**Comment by shouyou10** [comment]
I did not understand the meaning. So, I selected every folder. But that did not work out. Generally, in other apps when I press "Allow access to 'SD Card' ". The process completes there.

**Comment by Alfavio** [comment]
Here is the video tutorial to do this. It helped you, or the issue persists?

https://user-images.githubusercontent.com/5478075/110198093-3d8bfd00-7e50-11eb-84cd-1b3235f7b3d0.mp4

**Comment by shouyou10** [comment]
Thank you for the video. Unfortunately the problem persists. Allow me to explain. In the picture:1, the file manager (Material Files) have the access of my SD card. In the picture:2, it may look like Secure File Manager does not have the access and therefore not showing those folders of SD card. The reality is different. As you can see in the picture:3, I created a new folder: 'New Folder'. In the picture:4, nothing present in Secure File Manager. But as you can clearly see in picture:5, a new folder: 'New Folder' has been created. The main issue is I can neither see nor access this folder or any other folders of the SD card on Secure File Manager.
![Imagepipe_0](https://user-images.githubusercontent.com/76897897/110322120-8b4c6500-800a-11eb-906b-e85f510f27e0.jpg)
![Imagepipe_1](https://user-images.githubusercontent.com/76897897/110322128-8d162880-800a-11eb-88f8-71efa30531b7.jpg)
![Imagepipe_2](https://user-images.githubusercontent.com/76897897/110322131-8daebf00-800a-11eb-99d6-434360ac3e65.jpg)
![Imagepipe_3](https://user-images.githubusercontent.com/76897897/110322134-8e475580-800a-11eb-9495-b285a88251df.jpg)
![Imagepipe_4](https://user-images.githubusercontent.com/76897897/110322138-8edfec00-800a-11eb-91b6-f05229d42035.jpg)

**Comment by Alfavio** [comment]
I tested it on 4 different devices and it works. It looks like the app has a problem with your device. May I ask you, what are your device model and an Android version? I'm going to try to test the app on your device model via [BrowserStack](https://www.browserstack.com/).

**Comment by shouyou10** [comment]
Sure. My device is 'Samsung Galaxy A30'. Model: SM-A305F/DS. Android version: 10. Thank you for asking.

**Comment by Alfavio** [comment]
I tried testing the app via BrowserStack, but the Samsung Galaxy A30 is not supported, so I tested the app on the Samsung Galaxy A50. A device in the BrowserStack has not inserted SD cards, so now I can't reproduce your bug and fix it. Now, I have no clue how to fix this bug without physical access to the phone of your model.

_Note: I noticed, that on the Samsung devices the app detects an SD card if the SD card is not inserted in the device._
